CHANGELOG.md

```diff
@@ -1,3 +1,139 @@
+### citus v10.0 (February 16, 2021) ###
+
+* Adds shard rebalancer
+
+* Adds `alter_distributed_table` and `alter_table_set_access_method` UDFs
+
+* Adds `alter_old_partitions_set_access_method` UDF to compress old partitions
+
+* Adds cascade option to `undistribute_table` UDF for foreign key connected
+  relations
+
+* Adds support for local distributed/subquery joins
+
+* Allows distributed table creation immediately after CREATE EXTENSION citus
+
+* Automatically adds coordinator to the metadata before the first distributed
+  table created
+
+* Enables foreign keys between reference tables and local tables
+  (`citus.enable_local_reference_table_foreign_keys`)
+
+* Introduces adaptive connection management for local nodes
+
+* Adds `pg_catalog.citus_shards`, `pg_catalog.citus_shards_on_worker` and
+  `pg_catalog.citus_shard_indexes_on_worker` views
+
+* Adds `pg_catalog.time_partitions` view for simple (time) partitions and their
+  access methods
+
+* Adds `public.citus_tables` view
+
+* Adds `remove_local_tables_from_metadata` UDF for local tables automatically
+  added to metadata when defining foreign keys with reference tables
+
+* Adds support for `CREATE INDEX` commands without index name on citus tables
+
+* Adds support for `CREATE TABLE ... USING table_access_method` statements
+
+* Adds support for co-located/recurring correlated subqueries
+
+* Adds support for co-located/recurring sublinks in the target list
+
+* Adds support for local execution in `INSERT..SELECT` commands that perform
+  re-partitioning
+
+* Propagates `ALTER TABLE .. SET LOGGED/UNLOGGED` statements
+
+* Propagates `ALTER INDEX .. SET STATISTICS` statements
+
+* Propagates `ALTER SCHEMA RENAME` statements
+
+* Propagates `CREATE STATISTICS` statements across workers.
+
+* Propagates `DROP STATISTICS` statements across the workers
+
+* Propagates all types of `ALTER STATISTICS` statements.
+
+* Propagates columnar table settings for distributed tables
```

Comment:
I'm not sure we need `Propagates columnar table settings for distributed tables` as that is part of columnar anyway?

Reply:
yeah, I forgot to ask but I agree that we should remove that